pages/search/search.js

```diff
@@ -25,19 +25,15 @@ module.exports = settings => {
     res.locals.static.content = merge({}, res.locals.static.content, content(req.searchType));
     res.locals.static.profile = req.user.profile;
     res.locals.static.searchType = req.searchType;
-    res.locals.static.searchableModels = req.query.searchableModels;
     next();
   });
 
   app.use(datatable({
     configure: (req, res, next) => {
       const searchType = req.searchType;
-
       req.datatable.searchType = searchType;
-      set(req.datatable, 'filters.active', { status: ['active'] });
```

Comment:
This was the main culprit of the paging not working.

Getting it to default to active on some of the tabs is now done in the component itself.